src/qililab/qprogram/qblox_compiler.py

```diff
@@ -63,6 +64,7 @@ def __init__(self) -> None:
         self.qpy_sequence = QPy.Sequence(
             program=QPy.Program(), waveforms=QPy.Waveforms(), acquisitions=QPy.Acquisitions(), weights=QPy.Weights()
         )
+        # self.qpy_sequence._program.blocks[0].append_component(QPyInstructions.SetMrk(15))
```

Comment:
Is this comment necessary? like a reminder or something?

Reply:
No I forgot it there, I will remove it.